README.md

```diff
@@ -119,9 +119,9 @@ parse it's contents.
 
 ```
 usage: cchecker.py [-h] [--test TEST] [--criteria [{lenient,normal,strict}]]
-                   [--verbose] [--skip-checks SKIP_CHECKS]
+                   [--verbose] [--describe-checks] [--skip-checks SKIP_CHECKS]
                    [-f {text,html,json,json_new}] [-o OUTPUT] [-V] [-l]
-                   [-d DOWNLOAD_STANDARD_NAMES]
+                   [-d [DOWNLOAD_STANDARD_NAMES]]
```

Comment:
Will need to fix this `[DOWNLOAD_STANDARD_NAMES]` -> `DOWNLOAD_STANDARD_NAMES` to indicate not optional as well.